src/pages/Product/Product.js

```diff
@@ -1,7 +1,26 @@
 import React from "react";
+import { useState } from "react";
```

Comment:
```suggestion
import React, { useState }  from "react";
```
같은 라이브러리라면 이렇게 가져올 수 있습니다